appnotes/tensorflow.markdown

```diff
@@ -143,25 +161,19 @@ The module can hence be loaded with:
 ```bash
 {{site.data.terminal.prompt}} module purge
 {{site.data.terminal.prompt}} module load GCC/9.3.0  OpenMPI/4.0.3
-{{site.data.terminal.prompt}} module load nvidia-tensorflow/1.15.5+nv21.10-Python-3.8.2
+{{site.data.terminal.prompt}} module load nvidia-tensorflow/1.15.5+nv22.01-Python-3.8.2
 ```
 
-Note that this also imports CUDA into your environment automatically. 
+Note that this also imports CUDA into your environment automatically.
 
-Installing TensorFlow 1.15 from PyPi using pip is not advisable. The PyPi builds are based on cuDNN 7 and give significantly worse performance. 
+Installing TensorFlow 1.15 from PyPi using pip is not advisable. The PyPi builds are based on cuDNN 7 and give significantly worse performance.
 
 For example, in a ResNet50 classification benchmark:
 
-| Build                       | fp 32 (images/second) | fp 16 (images/second) |
-|-----------------------------|-----------------------|-----------------------|
-| PyPi build        (cuDNN 7) |        625            |         N/A           |
-| nvidia-tensorflow (cuDNN 8) |        1027           |        2486           |
+| Build                               | fp 32 (images/second) | fp 16 (images/second) |
+|-------------------------------------|-----------------------|-----------------------|
+| PyPi build                (cuDNN 7) |        625            |         N/A           |
+| nvidia-tensorflow/nv21.10 (cuDNN 8) |        1027           |        2486           |
```

Comment:
I'm not sure I'd both with this line, I think just listing nvidia-tensorflow/nv22.01 is sufficient.

Reply:
Good, let me edit it right here